feature/HW-16/index.js

```diff
@@ -0,0 +1,8 @@
+// let time = prompt('Введите количество часов');
+// let WaterLiters;
+// WaterLiters = Math.floor(time * 0.5);
+// alert(`Выпито литров воды: ${WaterLiters}`);
+
+
+let year = prompt('Введите год:')
+let stoletie;
```

Comment:
Транслит не рекомендуется.
```suggestion
let century;
```